hexrdgui/masking/mask_border_style_picker.py

```diff
@@ -0,0 +1,60 @@
+from PySide6.QtCore import QObject
+from PySide6.QtGui import QColor
+from PySide6.QtWidgets import QColorDialog
+
+from hexrdgui.ui_loader import UiLoader
+
+
+class MaskBorderStylePicker(QObject):
+
+    def __init__(self, original_color, original_style, original_width, parent=None):
+        super().__init__(parent)
+
+        loader = UiLoader()
+        self.ui = loader.load_file('mask_border_style_picker.ui', parent)
+        self.original_color = original_color
+        self.original_style = original_style
+        self.original_width = original_width
+        self.color = original_color
+        self.style = original_style
+        self.width = original_width
+
+        self.setup_ui()
+        self.setup_connections()
+
+    def setup_ui(self):
+        self.ui.border_color.setText(self.original_color)
+        self.ui.border_color.setStyleSheet('QPushButton {background-color: %s}' % self.original_color)
```

Comment:
```suggestion
        self.ui.border_color.setStyleSheet(f'QPushButton {{background-color: {self.original_color}}}')
```

No need to change this, but in case you were curious, this is how you'd do this with an f-string (double braces get transformed into a literal single brace).